plugins/git/git.plugin.zsh

```diff
@@ -206,159 +129,213 @@ function ggu() {
+alias gcor='git checkout --recurse-submodules'
+alias gcb='git checkout -b'
+alias gcd='git checkout $(git_develop_branch)'
+alias gcm='git checkout $(git_main_branch)'
+alias gco='git checkout'
+alias gcpa='git cherry-pick --abort'
+alias gcpc='git cherry-pick --continue'
+alias gcp='git cherry-pick'
+alias gclean='git clean --interactive -d'
+alias gcl='git clone --recurse-submodules'
+alias gcam='git commit --all --message'
+alias gcasm='git commit --all --signoff --message'
+alias gcas='git commit --all --signoff'
+alias gcssm='git commit --gpg-sign --signoff --message'
+alias gcss='git commit --gpg-sign --signoff'
+alias gcs='git commit --gpg-sign'
+alias gcmsg='git commit --message'
+alias gcsm='git commit --signoff --message'
+alias gca!='git commit --verbose --all --amend'
+alias gcan!='git commit --verbose --all --no-edit --amend'
+alias gcans!='git commit --verbose --all --signoff --no-edit --amend'
+alias gca='git commit --verbose --all'
+alias gc!='git commit --verbose --amend'
+alias gcn!='git commit --verbose --no-edit --amend'
+alias gc='git commit --verbose'
+alias gcf='git config --list'
+alias gdct='git describe --tags $(git rev-list --tags --max-count=1)'
+alias gdcw='git diff --cached --word-diff'
+alias gdca='git diff --cached'
+alias gds='git diff --staged'
+alias gdw='git diff --word-diff'
+alias gdup='git diff @{upstream}'
+alias gdt='git diff-tree --no-commit-id --name-only -r'
+alias gd='git diff'
+alias gfo='git fetch origin'
+alias gf='git fetch'
+alias gga='git gui citool --amend'
+alias gg='git gui citool'
+alias ghh='git help'
 alias glgga='git log --graph --decorate --all'
 alias glgm='git log --graph --max-count=10'
-alias glo='git log --oneline --decorate'
-alias glol="git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset'"
-alias glols="git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset' --stat"
-alias glod="git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset'"
-alias glods="git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset' --date=short"
-alias glola="git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset' --all"
-alias glog='git log --oneline --decorate --graph'
+alias glod='git log --graph --pretty="%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset"'
+alias glods='git log --graph --pretty="%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset" --date=short'
+alias glola='git log --graph --pretty="%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset" --all'
+alias glols='git log --graph --pretty="%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset" --stat'
+alias glol='git log --graph --pretty="%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset"'
```

Comment:
I changed the order of the quotation marks `" '` to `' "`. Better for Google Sheets.

Reply:
if it doesn't change the functionality (like there's some variable or something like that it should be fine 👍🏻